Explorer/Assets/DCL/Ipfs/LogIpfsRealm.cs

```diff
@@ -77,6 +77,18 @@ public URLDomain EntitiesActiveEndpoint
             }
         }
 
+        public URLDomain AssetBundleRegistry
+        {
+            get
+            {
+                var result = origin.AssetBundleRegistry;
+                ReportHub
+                    .WithReport(ReportCategory.REALM)
+                    .Log($"IpfsRealm AssetBundleRegistry requested, result: {result}");
+                return result;
+            }
+        }
```

Comment:
Property getters should not have side effects. The log message suggests that there are side effects.